full-service/src/db/schema.rs

```diff
@@ -116,6 +115,7 @@ diesel::table! {
 
 diesel::joinable!(assigned_subaddresses -> accounts (account_id));
 diesel::joinable!(authenticated_sender_memos -> txos (txo_id));
+diesel::joinable!(destination_memos -> txos (txo_id));
```

Comment:
Running the migration picked up on this change - seems legit, and like it should be in here